typedb/concept/proto/concept_proto_reader.py

```diff
@@ -84,7 +84,7 @@ def thing_type(proto_type: concept_proto.Type):
     elif proto_type.encoding == concept_proto.Type.Encoding.Value("ATTRIBUTE_TYPE"):
         return attribute_type(proto_type)
     elif proto_type.encoding == concept_proto.Type.Encoding.Value("THING_TYPE"):
-        return _ThingType(Label.of(proto_type.label), proto_type.root)
+        return _ThingType(Label.of(proto_type.label), proto_type.isRoot, proto_type.isAbstract)
```

Comment:
Are these really camel cased? This looks like a typo to me.
```suggestion
        return _ThingType(Label.of(proto_type.label), proto_type.is_root, proto_type.is_abstract)
```
Of course, we know the properties of the root thing type, so this is also an option ...
```suggestion
        return _ThingType(label = Label.of(proto_type.label), is_root = True, is_abstract = True)
```

Reply:
Looks like this same typo has been made in a bunch of other places in the PR.

Reply:
is actually correct :D

Reply:
@flyingsilverfin sorry, I should chased this down harder. Root cause was a bug in `typedb_protocol`. fixed by:
- https://github.com/vaticle/typedb-protocol/pull/175